com.bitwig.BitwigStudio.yaml

```diff
@@ -0,0 +1,75 @@
+---
+app-id: com.bitwig.BitwigStudio
+command: bitwig-studio
+runtime-version: "20.08"
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+rename-desktop-file: bitwig-studio.desktop
+rename-icon: bitwig-studio
+#rename-appdata-file: 'bitwig-studio.appdata.xml'
+finish-args:
+  # TODO: we need MIDI, Audio, DRI, and realtime performances
+  - --device=all
+  - --nodevice=kvm
+
+  # Needed to talk with X11, Wayland, pulseaudio and pipewire
+  - --share=ipc
+  - --filesystem=xdg-run/pipewire-0
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+
+  # Needed for updating the sound content, checking for online notifications and authentication
+  - --share=network
+
+  # For desktop notifications
+  - --talk-name=org.freedesktop.Notifications
+
+  # Give access to HOME
+  - --filesystem=home
```

Comment:
Would it make sense to limit filesystem access to xdg-music, xdg-download, /media and /run/media?

Reply:
No, because we load plugins and we don't know they're requirements.

Reply:
Do you suggest to drop -socket=wayland ?
Why?

Reply:
it would need `~/.BitwigStudio` and `~/Bitwig\ Studio` in `--persist

Reply:
@hfiguiere I'm not sure if whitespace needs escaping here.

Reply:
It has access to home, so it is not necessary, is it?

Reply:
@abique Yeah, if I'm not mistaken, `--persist` is no-op when app has home access, but it would ensure the app keeps working if the the user decides to revoke home access. I've assumed you have `--persist=~/.java` for this reason.

Reply:
OK

Reply:
I was mentioning it in case it was decided to restric --filesystem. That's it.